src/types/publication.types.ts

```diff
@@ -156,9 +151,8 @@ export interface Doi {
 }
 
 export const emptyPublication: Publication = {
-  modified: '', // date?
```

Comment:
modifiedDate finnes jo som felt i datamodellen

Reply:
modellen har flere felter som vi atm ikke har noen plass å vise. Per nå bruker vi `date` som ligger i `entityDescription` og `createdDate` (for mine publikasjoner). Jeg tenker det er greit å ta de inn gradvis som vi får bruk for de. Ganske sannsynlig at vi vil vise lastModified en plass også da... what u thinks?

Reply:
i thinks that du har sikkert rett. vi tar det inn gradvis. 

men! createdDate og modifiedDate blir satt i backend, så egentlig burde vi hatt et interface med felter som kan redigeres og et interface med felter som kun er readonly. evt prefix'e de som er readonly med readonly? 
https://www.logicbig.com/tutorials/misc/typescript/interfaces-with-read-only-properties.html

Reply:
kunne evt hatt en `FormikPublication` og en `Publication`?

Reply:
oi, visste ikke at TS hadde readonly engang 🤦‍♂ men liker den løsningen veldig godt!

Reply:
readonly ~kunne~ burde nok vært brukt på veldig mange andre types også. trolig så godt som alt som kommer fra backend

Reply:
ja, vi får huske på det fremover 😅